src/containers/WebComponentLoader.jsx

```diff
@@ -191,11 +190,27 @@ const WebComponentLoader = (props) => {
         </Style>
       </SettingsContext.Provider>
     </>
-  ) : (
+  );
+
+  const renderFailedState = () => (
+    <>
+      <p>{t("webComponent.failed")}</p>
+    </>
+  );
+
+  const renderLoadingState = () => (
     <>
       <p>{t("webComponent.loading")}</p>
     </>
   );
+
+  if (loading === "success" && project?.identifier) {
+    return renderSuccessState();
+  } else if (["idle", "failed"].includes(loading) && !project?.identifier) {
+    return renderFailedState();
+  } else {
+    return renderLoadingState();
+  }
```

Comment:
💪 YESSSS!! No more `loading...` when its failed 😍 I'd love to whack the loader here too

Reply:
yeh, it was making things very confusing that it just sat on loading...I'm going to stop short of adding the loader or I'll never get this tied up, but would be awesome to get that in place before long too